Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -26,8 +26,12 @@ configuration:
   name: incidents_per_fetch
   required: false
   type: 0
-- display: Fetch query
-  name: fetch_query
+- display: Detections fetch query
+  name: detections_fetch_query
+  required: false
+  type: 0
+- display: Incidents fetch query
+  name: incidents_fetch_query
```

Comment:
I consulted with @Arsenikr about this, and in his opinion, it's just changing the name of the argument, besides it does not change anything.

Reply:
@ronykoz Would love your opinion here

Reply:
no need to change the name, just the display name it might cause BC

Reply:
@ronykoz @guykeller 
To - `fetch query` and `incidents fetch query`?

When I added the option to do fetch for incidents and not just detections I did not add fetch query for the fetch incidents because we did not know it was different.
And it turns out they have a different query, so I added fetch query to fetch incidents as well.

Reply:
i mean that we should keep `incidents_fetch_query` as `fetch_query` in-order to not break BC

Reply:
@ronykoz Are you sure that changing the display name can't cause BC?

Reply:
yes